doc/includes/RECIPES.md

```diff
@@ -778,3 +778,232 @@ explain select * from "Hero" where details#>>'{type}' = 'Hero';
  Index Scan using "Hero_expr_idx" on "Hero"
    Index Cond: ((details #>> '{type}'::text[]) = 'Hero'::text)
 ```
+
+## Ternary relationships
+
+Assume we have the following Models:
+
+1. user `(id, first_name, last_name)`
+1. group `(id, name)`
+1. permission `(id, label)`
+1. user_group_permission `(user_id, group_id, permission_id, extra_attribute)`
+
+Here's how you could create your models:
+
+```js
+// User.js
+const { Model } = require("objection");
+
+class User extends Model {
+  static get tableName() { return "user"; }
+  static get relationMappings() {
+    return {
+      groups: {
+        relation: Model.ManyToManyRelation,
+        modelClass: require("Group"),
```

Comment:
This should be require('./Group') or some other relative path. Not super important for this example, but it's better to be precise